xds/internal/balancer/edsbalancer/eds_impl_test.go

```diff
@@ -56,6 +56,36 @@ func init() {
 		testEndpointAddrs = append(testEndpointAddrs, fmt.Sprintf("%d.%d.%d.%d:%d", i, i, i, i, i))
 	}
 	balancergroup.DefaultSubBalancerCloseTimeout = time.Millisecond
+	clusterimpl.NewRandomWRR = testutils.NewTestWRR
+	weightedtarget.NewRandomWRR = testutils.NewTestWRR
+	balancergroup.DefaultSubBalancerCloseTimeout = time.Millisecond * 100
```

Comment:
Is it common to use `time.Millisecond * 100`? I have only ever seen `100 * time.Millisecond`

Reply:
I think we have both. I'm leaving it as is, unless you have a strong feeling of it.

```
❯ rg "\* time.Millisecond" | wc -l
139

❯ rg "time.Millisecond \*" | wc -l
28
```